Source/ablastr/fields/PoissonSolver.H

```diff
@@ -263,10 +263,15 @@ computePhi (amrex::Vector<amrex::MultiFab*> const & rho,
         mlmg.setVerbose(verbosity);
         mlmg.setMaxIter(max_iters);
         mlmg.setAlwaysUseBNorm(always_use_bnorm);
+        if (WarpX::grid_type == GridType::Collocated) {
```

Comment:
Oops, this slipped through. We cannot use WarpX objects in the ABLASTR library. WarpX depends on ABLASTR but not the other way around. Using this here breaks ImpactX.

I will post a PR to fix this.

Reply:
https://github.com/ECP-WarpX/WarpX/pull/5106